hll.c

```diff
@@ -2756,11 +2756,19 @@ setup_multiset(MemoryContext rcontext)
     MemoryContext oldcontext;
     multiset_t * msp;
 
+#if (PG_VERSION_NUM >= 110000)
+    tmpcontext = AllocSetContextCreateExtended(rcontext,
+                                               "multiset",
+                                               ALLOCSET_DEFAULT_MINSIZE,
+                                               ALLOCSET_DEFAULT_INITSIZE,
+                                               ALLOCSET_DEFAULT_MAXSIZE);
+#else
```

Comment:
That's the wrong fix FWIW, you should just replace the arguments with ALLOCSET_DEFAULT_SIZES. That works for 9.6+.   By using AllocSetContextCreateExtended you skip some error checks.